cmd/journald_test.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"context"
+	"os/user"
+	"strconv"
+	"testing"
+)
+
+func mockCheckDirFn() fn {
+	return func(s string, u uint32, m map[string]uint32) error {
+		return nil
+	}
+}
+
+func TestJournalCheck(t *testing.T) {
```

Comment:
I think we should have a test that checks the failure condition, too, so, if `fn` return a non-nil error, we want to check the output of the check in that case as well.